utils/manifest-utils/src/golem_certificate.rs

```diff
@@ -0,0 +1,96 @@
+use url::Url;
+use ya_client_model::NodeId;
+
+pub struct GolemCertificate {
+    pub node_id: NodeId,
```

Comment:
We have separate structure for `node.json` it is not the same as certificate

Reply:
I am not sure what was intention of naming it that way. Maybe @evik42 could you clarify?

From negotiations perspective: `GolemCertificate` + `some other data` = `node.json`
We get `node.json` from demand, then we pass it to `verify_golem_certificate` function, and we get `GolemCertificate`.